llm_toolkit/prompt_builder.py

```diff
@@ -115,9 +112,11 @@ class DefaultTemplateBuilder(PromptBuilder):
 
   def __init__(self,
                model: models.LLM,
+               benchmark: Optional[Benchmark] = None,
                template_dir: str = DEFAULT_TEMPLATE_DIR):
     super().__init__(model)
     self._template_dir = template_dir
+    self.benchmark = benchmark
```

Comment:
Thanks @DavidKorczynski ! I also wanted to make this change for sooo long, it's great that you did it!